app/src/main/java/com/taylorgirard/comicconvo/activities/MainActivity.java

```diff
@@ -44,6 +51,8 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
             }
         });
         //Set default
-        bottomNavigationView.setSelectedItemId(R.id.action_profile);
+        bottomNavigationView.setSelectedItemId(R.id.action_map);
```

Comment:
Yes, I was changing the default pages depending on whatever I was working on so that I could test more quickly. Final default page will be the match page I think